src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -667,6 +676,56 @@ public float AudioPitch
             }
         }
 
+        /// <summary>
+        /// Gets or sets the default codec type of the audio decoder.
+        /// </summary>
+        /// <value>A <see cref="CodecType"/> that specifies the type.
+        /// The default codec type could be different depending on the device capability.</value>
+        /// <remarks>
+        /// <para>To set, the player must be in the <see cref="PlayerState.Idle"/> state.</para>
+        /// <para>If H/W audio codec type is not supported in some cases, S/W audio codec type could be used instead.</para>
+        /// <para>The availability could be changed depending on the codec capability.
+        /// If an application wants to use the H/W audio codec type as default,
+        /// following functions should be called after the codec type is set. :<br/>
```

Comment:
Suggestion:
        /// The following functions should be called after the codec type is set. :<br/>